public/themes/default.css

```diff
@@ -351,7 +350,7 @@ video.post-attachment {
   border-radius: 6px;
   background-color: var(--colors-background);
   max-width: 100%;
-  max-height: 384px;
+  max-height: 384px; 
```

Comment:
@lachlanjc I think the weird padding is caused by this one, because the `next/image` thing has it's own special way of doing image heights with padding